common/changes/@microsoft/rush/fixWorkspaceInstallSkip_2020-07-15-23-04.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Consider package.json when determining if install can be skipped for workspaces",
```

Comment:
We should clarify that you mean `PNPM workspaces`; later when we support Yarn this won't be obvious